scripts/dev/migrate-status-to-features

```diff
@@ -0,0 +1,88 @@
+import re
```

Comment:
```suggestion
#!/usr/bin/env python

import re
```

I don't think it would fix anything, but it should enable syntax highlighting on Github ;)